CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ target_link_libraries(roboteam_utils
 target_compile_options(roboteam_utils PRIVATE
         --std=c++17
         -Wall
-        -Werror
+#        -Werror
```

Comment:
Is this because of nlohnmann or because of proto?

Reply:
proto